metaflow/plugins/argo/argo_client.py

```diff
@@ -333,12 +333,15 @@ def schedule_workflow_template(self, name, schedule=None, timezone=None):
                 json.loads(e.body)["message"] if e.body is not None else e.reason
             )
 
-    def register_sensor(self, name, sensor=None):
+    def register_sensor(self, name, sensor=None, sensor_namespace=None):
         if sensor is None:
             sensor = {}
+        if sensor_namespace is None:
+            sensor_namespace = ARGO_EVENTS_SENSOR_NAMESPACE
```

Comment:
yea, I can do that.  I think that I have it this way because the original namespace parameter in the `__init__` method is a null value.  But I can very easily make this the default.